mac/ide-tour.md

```diff
@@ -110,7 +110,7 @@ For more information on using version control in Visual Studio, see the [Version
 ## Next steps
 
 - [Install Visual Studio for Mac](installation.md)
-- [Review the available workloads](/visualstudio/mac/workloads/)
+- [Review the available workloads](/visualstudio/mac/workloads.md)
```

Comment:
It actually needs to be:
**[Review the available workloads](/mac/workloads.md)**
Because the working link is: https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/mac/workloads.md without the "**/visualstudio**".